app/src/main/java/org/simple/clinic/activity/placeholder/PlaceholderScreen.kt

```diff
@@ -1,50 +1,54 @@
 package org.simple.clinic.activity.placeholder
 
-import android.content.Context
-import android.util.AttributeSet
-import android.widget.RelativeLayout
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.VISIBLE
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.Disposable
 import kotlinx.parcelize.Parcelize
-import org.simple.clinic.R
 import org.simple.clinic.await.Await
 import org.simple.clinic.await.Checkpoint
 import org.simple.clinic.databinding.ScreenPlaceholderBinding
-import org.simple.clinic.router.screen.FullScreenKey
+import org.simple.clinic.navigation.v2.ScreenKey
 import java.util.concurrent.TimeUnit.SECONDS
 
-class PlaceholderScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context, attrs) {
+class PlaceholderScreen : Fragment() {
   private val delayToShowMessage = SECONDS.toMillis(3).toInt()
   private val await = Await(listOf(Checkpoint.unit(delayToShowMessage)))
   private var awaitDisposable: Disposable? = null
-  private var binding: ScreenPlaceholderBinding? = null
+
+  private var _binding: ScreenPlaceholderBinding? = null
```

Comment:
Did we add to separate variables for binding because we didn't want to add `non-nullable` every time we called it? Or is there some other reason for this?

Reply:
Yep, it's to avoid using nullable variable